app/logic/events.py

```diff
@@ -66,34 +67,33 @@ def deleteEvent(eventId):
         else:
             createActivityLog(f"Deleted a non-program event, \"{event.name}\", which had a start date of {datetime.strftime(event.startDate, '%m/%d/%Y')}.")
 
-        Event.update({Event.deletionDate: datetime.now(), Event.deletedBy: g.current_user}).where(Event.id == event.id).execute()
-
+        Event.update({Event.deletionDate: datetime.now(), Event.deletedBy: g.current_user}).where(Event.id == event.id).execute()       
 
 def deleteEventAndAllFollowing(eventId):
         """
-        Deletes a recurring event and all the recurring events after it.
-        Modified to also apply to the case of events with multiple offerings
+        Deletes an event in the series and all events after it
+        
         """
         event = Event.get_or_none(Event.id == eventId)
         if event:
-            if event.recurringId:
-                recurringId = event.recurringId
-                recurringSeries = list(Event.select(Event.id).where((Event.recurringId == recurringId) & (Event.startDate >= event.startDate)))
-                deletedEventList = [recurringEvent.id for recurringEvent in recurringSeries]                
-                Event.update({Event.deletionDate: datetime.now(), Event.deletedBy: g.current_user}).where((Event.recurringId == recurringId) & (Event.startDate >= event.startDate)).execute()
-                return deletedEventList
-
-def deleteAllRecurringEvents(eventId):
+            if event.seriesId:
+                    seriesId = event.seriesId
+                    eventSeries = list(Event.select(Event.id).where((Event.seriesId == seriesId) & (Event.startDate >= event.startDate)))
+                    deletedEventList = [event.id for event in eventSeries]                
+                    Event.update({Event.deletionDate: datetime.now(), Event.deletedBy: g.current_user}).where((Event.seriesId == seriesId) & (Event.startDate >= event.startDate)).execute()
+                    return deletedEventList
+
+def deleteAllEventsInSeries(eventId):
         """
-        Deletes all recurring events.
-        Modified to also apply for events with multiple offerings
+         Deletes all events in a series by getting the first event in the series and calling deleteEventAndAllFollowing().
+         
         """
         event = Event.get_or_none(Event.id == eventId)
         if event:
-            if event.recurringId:
-                recurringId = event.recurringId
-            allRecurringEvents = list(Event.select(Event.id).where(Event.recurringId == recurringId).order_by(Event.startDate))
-            eventId = allRecurringEvents[0].id
+            if event.seriesId:
+                seriesId = event.seriesId
+            allSeriesEvents = list(Event.select(Event.id).where(Event.seriesId == seriesId).order_by(Event.startDate))
+            eventId = allSeriesEvents[0].id
```

Comment:
This if statement is the same as it was before, but what happens if there is no seriesId? this line will fail. All of it should be in the if statement I think

Reply:
Resolved. Also added error handling if event does not exist